Guide/DynamicProgramming/README.md

```diff
@@ -0,0 +1,12 @@
+# Dynamic Programming Guide
+
+## Overview
+<p> Dynamic Programming - An algorithm that solves smaller problems that can
```

Comment:
@ASAAR I changed the definition. I included memorization into my definition because it is what makes Dynamic Programming different from Divide and Conquer. I went back and looked at Gelfond's definition for DP and it is half a page long. I felt like that was a bit too long and technical. So, I shortened it to its main points.